jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditorDialog.kt

```diff
@@ -25,10 +25,11 @@ val relativeTimeUnit = mapOf(
 class QueryEditorDialog(
     private val project: Project,
     private val lGroupName: String,
-    private val client: CloudWatchLogsClient
+    private val client: CloudWatchLogsClient,
+    private val defaultQueryEditorState: Boolean
```

Comment:
This name is really confusing, can it be changed to something more indicative of what it does?

Reply:
changed it to initialQueryEditorState